packages/core/src/BinaryDataManager/index.ts

```diff
@@ -51,11 +51,10 @@ export class BinaryDataManager {
 		const retBinaryData = binaryData;
 
 		if (this.managers[this.binaryDataMode]) {
-			return this.managers[this.binaryDataMode]
+			await this.managers[this.binaryDataMode]
```

Comment:
This may be a breaking change... not sure. Previously, returning early would prevent line 62 from executing (setting the data in the return object to that from the request buffer). Simply awaiting here fixes this.